packages/core-contracts/contracts/test/ownership/OwnableMock.sol

```diff
@@ -4,28 +4,9 @@ pragma solidity ^0.8.0;
 import "../../ownership/Ownable.sol";
 
 contract OwnableMock is Ownable {
-    address public override owner;
-    address public override nominatedOwner;
-
     constructor(address firstOwner) {
```

Comment:
Is this mock still needed or can it be avoided?

Reply:
removed!